.github/workflows/build-enzyme-and-wasm.yml

```diff
@@ -34,30 +39,30 @@ jobs:
         with:
           name: algovivo.linux.wasm
           path: build/algovivo.wasm
-  build-wasm-macos:
+  build-enzyme-and-wasm-macos:
     runs-on: macos-latest
     steps:
       - name: Install system dependencies
         run: |
-          brew install llvm@11 ninja
+          brew install llvm@12 ninja
```

Comment:
Workaround for https://github.com/Homebrew/brew/issues/15859 which causes `brew install llvm@11` to fail.